### DISCUSSION TITLE
SO for Department of Defense.  Centos7 or Ubuntu?

### DISCUSSION THREAD

**Discussion by TotieBash** [Security-Onion-Solutions/securityonion#9541] [Unsupported Versions]
This question is for SO users in DoD.  Is your SO base operating system Centos7 or Ubuntu?

The reason I ask is in the past Ubuntu has been frowned upon for OS of choice within the Department of Defense.  In which RHEL is the easy choice and the path of least resistance.  However, on cyber.mil website I now see a bunch of STIG for Ubuntu 18.04 and 20.04 which lead me to believe that DoD has soften the stance in regards to Canonical.  I always thought that SO on Centos7 would be the best choice but now I am having second thought and maybe Ubuntu is now a better choice.  Any insights would be appreciated.  Thanks...

**Comment by lock-wire** [comment]
Unfortunately, the best answer is "It depends on your organization and approving official."  There are some STIG configurations that break functionality and will require the organization and approving official to accept or mitigate the assessed risk in the context of that information environment.  Your organization's skill and knowledge in either operating system will play a part in implementing and maintaining STIG configurations and/or assessed risk mitigations.
I imagine that most DoD organizations use CentOS7 because that is what the ISO is built on.  Ubuntu installations utilize the network install procedure which maybe be challenging in certain air-gap situations.  
This was a great question.  I'm definitely looking forward to more input from others in the community.

**Reply by TotieBash** [reply]
Thanks @lock-wire for the input.  I keep reading the Ubuntu improvements and I am liking what I see.

The Ubuntu 20.04 (Pro and Adv) has achieved Common Criteria (EAL2) and FIPS 140.2:
https://ubuntu.com/security/certifications
https://ubuntu.com/security/certifications/docs

The STIG compliance is impressively simple:
 https://youtu.be/p59QUSX4U2A

With the above improvements I am gonna at least check it out and see how it will behave with SO.  The airgap network would definitely be an issue but challenge accepted.  I'm thinking apt-catcher-ng for the Ubuntu patches and I think I can still use the iso image to run soup and just decline the option for os patches.  We shall see.

I am still curious to hear if there are anyone out there running SO on Ubuntu for the Department of Defense.  Mainly curious if they had difficulty through the RMF process accreditation.  Or if they use Centos7 and went through RMF process and accredited.
